### PR TITLE
Base structure and guidance comments created.

### PR DESCRIPTION
**Pull request by jusk02**
Base structure and guidance comments created.

### CODE REVIEW

**Comment by jusk02** [comment]
[Base HTML wireframe, as a developer i need a clear division structure through the site to start building sections.](https://trello.com/c/9iZXQBUL/1-base-html-wireframe-as-a-developer-i-need-a-clear-division-structure-through-the-site-to-start-building-sections)

**Comment by jusk02** [comment]
After the talk we have yesterday about component css being isolated on a file per component i came around to implementing this but have come to appreciate this should not be necessary if we use BEM as our selected style guide suggests, therefore i still think this code chunk is done, please let me know if you still think differently and we can discuss it.

**Review by Sonny97 — APPROVED** [review]

**Review by Sonny97 — APPROVED** [review]

File: assets/dummy.dummy [hunk 1]
```diff
@@ -0,0 +1 @@
+remove this file once we have atleast one asset.
```
Comment (camigomezdev) [inline assets/dummy.dummy]: can you create a README.md for register this folder?

File: js/logic.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+/* Logic for page animations and simple interactions should be placed here
+   Any page or component specific logic should be placed in its own page/component js file.
+**/
```
Comment (camigomezdev) [inline js/logic.js]: should this file be named as index.js?

File: index.html [hunk 3]
```diff
@@ -7,6 +7,14 @@
     <title>Hostal las paisas</title>
 </head>
 <body>
-    
+    <section>
```
Comment (camigomezdev) [inline index.html:13]: could add nav section?

File: js/logic.js (hunk 2 above)
Comment (jusk02) [reply js/logic.js]: i agree, will change.

**Review by Sonny97 — APPROVED** [review]

**Review by camigomezdev — APPROVED** [review]

**Review by Sonny97 — COMMENTED** [review]
que tal si hacemos una clase para las interrupciones, no tengo bien claro como aplicarlos a manera de codigo

**Review by Sonny97 — APPROVED** [review]

**Comment by jhonatanrojasbastidas** [comment]
Hey i need the contact info, location and the social media info from hostal las paisas.

**Comment by jusk02** [comment]
> Hey i need the contact info, location and the social media info from hostal las paisas.

This request is not pertinent to this PR, it should be done in trello.

**Pull request merged**